march_hardware/src/PDOmap.cpp

```diff
@@ -6,129 +6,152 @@
 
 namespace march4cpp
 {
-PDOmap::PDOmap()
+std::unordered_map<IMCObjectName, IMCObject> PDOmap::all_objects =
 {
-  this->initAllObjects();
-}
+  { IMCObjectName::StatusWord, IMCObject(0x6041, 16) },
```

Comment:
If you're rewriting the mapping anyway, please add 0x6007 the object in which we can set the way we react to EtherCAT communication loss